core/src/database/scheduling/match/match.model.ts

```diff
@@ -16,6 +16,10 @@ export class Match extends BaseModel {
     @Field(() => Boolean, {defaultValue: false})
     isDummy: boolean;
 
+    @Column({default: false})
+    @Field(() => Boolean, {defaultValue: false})
+    isNcp: boolean;
```

Comment:
https://app.clickup.com/t/2pdz0e8

Reply:
I'm specifically deferring this to the next ticket *because* it would involve another DB repo injection, which the ticket above has a task to push those (and thus this) to helper services.